Cargo.toml

```diff
@@ -82,6 +82,7 @@ kanidmd_lib = { path = "./server/lib" }
 kanidmd_lib_macros = { path = "./server/lib-macros" }
 kanidm_lib_crypto = { path = "./libs/crypto" }
 kanidm_lib_file_permissions = { path = "./libs/file_permissions" }
+kanidm_lib_identity_verification_types = { path = "./libs/identity_verification_types" }
```

Comment:
I'm not sure we need another crate here, can't we use proto? Or do these types need to be buried into their respective parts? I think a whole crate here is the wrong way.

Reply:
yeah, as a matter of fact I thought I was using proto :p